xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -819,6 +865,26 @@ void CGUIDialogVideoInfo::OnGetFanart()
   Update();
 }
 
+void CGUIDialogVideoInfo::OnSetUserrating()
```

Comment:
Well probably also have to go back to the core changes then, as they were all using "userrating".